modules/ztp-ztp-building-blocks-factory.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-factory-install-clusters.adoc
+
+[id="ztp-ztp-building-blocks_{context}"]
+
+= Zero touch provisioning building blocks
```

Comment:
Our repo's guidelines show these crammed together just as assemblies are: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Would do that here for consistency's sake.

Reply:
I need to remove this file from this PR.